projects/ADuCM3029_demo_cn0428_cn0429/src/adi_m355_gas_sensor.h

```diff
@@ -0,0 +1,215 @@
+*      contributors may be used to endorse or promote products derived
+*      from this software without specific prior written permission.
+*    - The use of this software may or may not infringe the patent rights
+*      of one or more patent holders.  This license does not release you
+*      from the requirement that you obtain separate licenses from these
+*      patent holders to use this software.
+*    - Use of the software either in source or binary form, must be run
+*      on or directly connected to an Analog Devices Inc. component.
+*
+*THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+*INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT, MERCHANTABILITY AND FITNESS FOR A
+*PARTICULAR PURPOSE ARE DISCLAIMED.
+*
+*IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+*EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, INTELLECTUAL PROPERTY
+*RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+*BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+*STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+*THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************
+**/
+
+#ifndef ADI_M355_GAS_SENSOR_H_
+#define ADI_M355_GAS_SENSOR_H_
+
+#include "adi_gas_sensor.h"
+#include "adi_m355_gas_sensor_config.h"
+#include <drivers/i2c/adi_i2c.h>
+#include <drivers/spi/adi_spi.h>
+
+
+#define	GET_STATUS              	0x02
+#define SET_TEMPERATURE         	0x03
+#define READ_TEMPERATURE        	0x04
+#define SET_HUMIDITY            	0x05
+#define	READ_HUMIDITY           	0x06
+#define SET_MEAS_TIME_MS        	0x07
+#define READ_MEAS_TIME_MS			0x53
+#define START_MEASUREMENTS      	0x08
+#define STOP_MEASUREMENTS       	0x0A
+#define SET_TIA_GAIN            	0x0C
+#define READ_TIA_GAIN           	0x0E
+#define SET_VBIAS		        	0x10
+#define READ_VBIAS			       	0x12
+#define SET_VZERO_VOLTAGE       	0x14
+#define READ_VZERO_VOLTAGE      	0x16
+#define SET_SENSOR_TYPE         	0x18
+#define SET_SENSOR_SENS         	0x19
+#define READ_SENSOR_TYPE        	0x1A
+#define READ_SENSOR_SENS        	0x1B
+#define	SET_TEMP_COMP           	0x1C
+#define READ_TEMP_COMP          	0x1D
+#define RUN_EIS						0x1E
+#define READ_EIS_RESULTS			0x1F
+#define READ_AVG_LSB            	0x20
+#define READ_EIS_RESULTS_FULL		0x25
+#define READ_RAW_LSB            	0x26
+#define READ_AVG_PPB            	0x30
+#define READ_RAW_PPB            	0x40
+#define RUN_PULSE_TEST          	0x45
+#define READ_PULSE_TEST_RESULTS 	0x50
+#define SET_RLOAD					0x51
+#define READ_RLOAD					0x52
+#define READ_200R_RTIA_CAL_RESULT 	0x55
+#define SET_I2C_ADDRESS				0x80
+
+
+
+
+typedef enum {
+	READ = 0,
+	WRITE
+} eI2C_DIR;
+
+typedef enum {
```

Comment:
I don't think you actually use these enums.

Reply:
I think you're right. I'll try removing them.